src/dafny/FileIO/FileIO.md

```diff
@@ -0,0 +1,122 @@
+
```

Comment:
This will have to be merged with the existing README.md as you move it under the existing FileIO.

Reply:
The existing README.md is going away as it has been subsumed by FileIO.md. Just doing that in two steps.